layouts/docs/search-data.json

```diff
@@ -0,0 +1,10 @@
+{
+{{- $pages := (where .Site.Pages ".Params.noindex" "ne" true) -}}
```

Comment:
Delete `layouts/_default/search-data.json` so this shows up in git as a move?

Reply:
Ah, totally thought I had. Good catch, thanks.